pom.xml

```diff
@@ -239,6 +239,7 @@
                 <requireBackwardCompatibility implementation="org.semver.enforcer.RequireBackwardCompatibility">
                   <excludes>
                     <exclude>com/querydsl/core/types/dsl/PathBuilderValidator</exclude>
+                    <exclude>com/querydsl/sql/Union</exclude>
```

Comment:
The enforcer complained about the Union interface change. A superinterface was added.